File: Rally-BackEnd/backend/src/main/java/org/rally/backend/userprofilearm/model/ProfilePicture.java
package org.rally.backend.userprofilearm.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;


@Entity
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ProfilePicture {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String userId;
    private String type;
    @Column(columnDefinition = "MEDIUMBLOB NOT NULL")
    private byte[] image;

}

File: Rally-FrontEnd/src/app/user-profile-arm/models/ProfilePicture.ts
export class ProfilePicture {

	id: string;
    userId: string;
    type: string;
    image: string;
	
	constructor(id: string, userId: string, type: string, image: string){
		this.id = id;
		this.userId = userId;
        this.type = type;
        this.image = image;
	}
}